zmq_transport/server/zmq_server.py

```diff
@@ -1,38 +1,17 @@
 """
 ZeroMQ Server.
 """
-# ZeroMQ Imports
+
+
 import zmq
-from zmq.eventloop.zmqstream import ZMQStream
-from zmq.eventloop.ioloop import IOLoop, PeriodicCallback
-
-# Protobuf Imports
-from google.protobuf.message import DecodeError
-
-# Local Imports
-from zmq_transport.config import settings
-from zmq_transport.config.protobuf_settings import (
-    MSG_TYPE_PING,
-    MSG_TYPE_SYNC_TYPE,
-    MSG_TYPE_ZMQ_VERB,
-    MSG_TYPE_GET_SYNC_INFO_REQUEST,
-    MSG_TYPE_SEND_DOCUMENT_REQUEST,
-    MSG_TYPE_ALL_SENT_REQUEST,
-    MSG_TYPE_GET_DOCUMENT_REQUEST,
-    MSG_TYPE_PUT_SYNC_INFO_REQUEST
-)
+from zmq.eventloop.ioloop import IOLoop
+
```

Comment:
It's only stylistic choice to separate imports into blocks that import more or less from the same source. Not really needed, feel free to modify as you think it's best.